<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="startDialogWelcomeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to Pirkanmaa Conquest!</string>
   </property>
  </widget>
  <widget class="QLabel" name="startDialogInfoLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start/Load game:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startDialogNewButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDialogLoadButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Game</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loadFileEntry">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
